frontend/src/pages/books/BookAdd.tsx

```diff
@@ -150,71 +153,181 @@ export default function BookAdd() {
       field: "pageCount",
       header: "Page Count",
       style: { width: "2%", fontSize: "small" },
-      customBody: (rowData: BookWithDBTag) =>
-        NullableIntegerEditor(
-          rowData.pageCount,
-          (newValue) => {
-            setBooks((draft) => {
-              const book = findById(draft, rowData.id)!;
-              book.pageCount = newValue;
-            });
-          },
-          "integerNumbeBookAdd"
-        ),
+      customBody: (rowData: BookWithDBTag) => (
+        <>
+          <div>
```

Comment:
Maybe it you should extract these components during refactorization.

Reply:
Noted, I will do this when I do the refactor for updating the endpoint